installation/local/devVM.rst

```diff
@@ -2,26 +2,30 @@
 Local Installation
 ******************
 
-This guide explaines how to use the virtual machine to create test enviroment.
+This guide explains how to use the Virtual Machine to create a test environment.
 
 You can download the OVA image at the following link:
 
 OVA Image: `download <https://developers.italia.it/static/DAF-Ubuntu16-Docker-test.ova>`_ (8.6 GB)
 
+.. warning::
```

Comment:
I think it is important to mention this prerequisite. What do you think?

Reply:
Agreed. Hopefully we'll switch to Vagrant soon, maybe we should mention it? Or mention that we'd love PRs for this?

Reply:
OK, I will mention both things (Vagrant + we'd love PRs).